### PR TITLE
Configurable Spellchecker

### PR DESCRIPTION
**Pull request by felixrieseberg**
This is a big one: This commit makes the spellchecker configurable.
Users can choose their favorite language in the preferences, which will
then be used as a spellchecker in the blog.

As always, we’re using the current operating system’s spellchecker.

### CODE REVIEW

**Comment by felixrieseberg** [comment]
I also made sure that this thing runs well on Windows. I'd like to test it on Linux before merging, but passing tests give me confidence.

File: app/components/gh-instance-host.js [hunk 1]
```diff
@@ -196,5 +201,17 @@ export default Component.extend({
         if (this.get('blog')) {
             this.get('blog').updateName();
         }
+    },
+
+    /**
+     * Sends the current spellchecker language to the webview
+     */
+    _setupSpellchecker() {
+        let $webviews = this.$('webview');
```
Comment (bnookala) [inline app/components/gh-instance-host.js]: maybe we should keep a reference to this webview in other places

Comment (felixrieseberg) [inline app/components/gh-instance-host.js]: It's a good idea - I had some trouble (in the past) with rerenders and the references essentially going stale, but I'd <3 that.

File: app/components/gh-select-native.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    return this;
+}
+
+export default Component.extend({
+    content: null,
+    prompt: null,
+    optionValuePath: 'id',
+    optionLabelPath: 'title',
+    selection: null,
+    action: K, // action to fire on change
```
Comment (bnookala) [inline app/components/gh-select-native.js]: K?

Comment (felixrieseberg) [inline app/components/gh-select-native.js]: That's a weird self-referencing thing, but the whole class comes straight from Ghost, so I simply trust that our colleagues over there now what they're doing 👯

File: app/utils/iso639.js [hunk 3]
```diff
@@ -0,0 +1,778 @@
+/**
```
Comment (bnookala) [inline app/utils/iso639.js]: I'm surprised this isn't a node package by itself :0

Comment (felixrieseberg) [inline app/utils/iso639.js]: ...it totally is :/ For a brief moment, I thought I could maybe post this and help the world

Comment (bnookala) [inline app/utils/iso639.js]: actually, it does appear to be in npm as `languages`. Could we use this? https://www.npmjs.com/package/languages

**Pull request merged**